packages/neo-one-server-client/src/ServerManager.ts

```diff
@@ -166,11 +210,25 @@ export class ServerManager {
     return { pid: child.pid, started: true };
   }
 
-  public async writePID(pid: number): Promise<void> {
+  public async writeVersionPid(pid: number, version: string): Promise<void> {
+    await fs.ensureDir(path.dirname(this.pidPath));
```

Comment:
Remove

Reply:
Actually, you should be doing `ensureDir` on `path.dirname(this.versionPath)`

Reply:
I would just change this to two methods, `writePID` and `writeVersion` and then call both from this method `wirteVersionPID`.

Reply:
👍